src/components/notebook/NotebookViewer.tsx

```diff
@@ -185,6 +187,42 @@ export const NotebookViewer: React.FC<NotebookViewerProps> = ({
           Math.max(...cells.map((c: CellData) => c.position), -1) + 1;
       }
 
+      // Get default AI model if creating an AI cell
+      let aiProvider, aiModel;
+      if (cellType === "ai") {
+        // Try to get the last used AI model from notebook metadata
+        const lastUsedModelKey = metadata.find(
+          (meta: any) => meta.key === "lastUsedAiModel"
+        );
+        const lastUsedProviderKey = metadata.find(
+          (meta: any) => meta.key === "lastUsedAiProvider"
+        );
```

Comment:
These could be done as proper LiveStore queries (ahem, SQL queries with the ORM).